meerk40t/core/node/elem_text.py

```diff
@@ -1,10 +1,12 @@
 import re
 from copy import copy
-from math import tau
+from math import ceil, floor, tau
+
+from PIL import Image
```

Comment:
PIL might need to be done more internally. While it's fairly ubiquitous everything that isn't an image shouldn't require the Import. This makes it absolutely required to use anything in the CLI.